modules/cli-logging-in.adoc

```diff
@@ -52,4 +52,9 @@ Welcome! See 'oc help' to get started.
 <2> Enter whether to use insecure connections.
 <3> Enter the user's password.
 
+[NOTE]
+====
+If you are logged in to the {product-title} web console, you can generate an `oc login` command that includes your token and server information. You can use the command to log in to the {product-title} CLI without the interactive prompts. To generate the command, select *Copy login command* from the drop-down menu at the top right of the web console.
```

Comment:
I _think_ we generally refer to it simply as "the web console".

```suggestion
If you are logged in to the web console, you can generate an `oc login` command that includes your token and server information. You can use the command to log in to the {product-title} CLI without the interactive prompts. To generate the command, select *Copy login command* from the drop-down menu at the top right of the web console.
```

Reply:
@bobfuru thank you for your review and for highlighting this. I have now resolved this suggestion.